snapcraft/plugins/plainbox_provider.py

```diff
@@ -43,6 +44,13 @@ def __init__(self, name, options, project):
 
     def build(self):
         super().build()
+        env = os.environ.copy()
+        provider_stage_dir = os.path.join(self.project.stage_dir, "providers")
```

Comment:
oh, I see there is a combination of single and double quotes in this function. Now that you are touching this code, I would appreciate if you update all of them to single quotes :)